modules/backend/formwidgets/repeater/partials/_repeater_item.htm

```diff
@@ -12,8 +12,7 @@
                 data-request="<?= $this->getEventHandler('onRemoveItem') ?>"
                 data-request-data="'index': '<?= $indexValue ?>'"
                 data-request-success="$(this).closest('.field-repeater-item').remove()"
-                data-request-confirm="<?= e(trans('backend::lang.form.action_confirm')) ?>"
-                data-request-complete="($('.field-repeater').attr('data-max-items') > $('.field-repeater-item').length || $('.field-repeater').attr('data-max-items') == 0) ? $('.field-repeater-add-item').show() : $('.field-repeater-add-item').hide()">
+                data-request-confirm="<?= e(trans('backend::lang.form.action_confirm')) ?>">
```

Comment:
I try to listen from repeater.js when this event get triggered using global events http://octobercms.com/docs/ajax/javascript-api#global-events but it does not work